File: Scripts/laptop/regular-expressions/perl/regular-expressions.pl
my $r = 1;

sub executeTask {
	my ($i) = @_;	
	$string = "this is a matching string";
	if ($string =~ /string$/) {}
	if ($string =~ s/ string/ istyfied/) {}
	return ($i + 1);
}


for( $a=0; $a < 10000000; $a = $a + 1) {
	$r = executeTask($a + $r);
}

File: Scripts/laptop/regular-expressions/python/regular-expressions.py
import re

def executeTask(i):
    string = "This is a matching string"
    re.search('string$', string)
    string = re.sub("string", "istyfied", string)
    return i + 1

r = 1
for i in range(10000000):
	r = executeTask(i + r)
